app-text/ebook-tools/ebook-tools-0.2.2-r1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=4
+
+inherit cmake-utils multilib
+
+DESCRIPTION="Tools for accessing and converting various ebook file formats"
+HOMEPAGE="http://sourceforge.net/projects/ebook-tools"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~x86 ~amd64-fbsd"
+IUSE="lit2epub"
```

Comment:
Then there would be no reason for a revbump as no behaviour would change.

Pick one:
1. revbump and `IUSE="lit2epub"`
2. no revbump and `IUSE="+lit2epub"`